lrlog/lrlog.go

```diff
@@ -0,0 +1,143 @@
+package lrlog
```

Comment:
??

Reply:
This logging package is used by heartbeat package